src/javascript/app_2/pages/statement/statement.jsx

```diff
@@ -2,44 +2,16 @@ import React              from 'react';
 import moment             from 'moment';
```

Comment:
This refactoring doesn't really relate to this PR, "Settings page". 

Maybe can you create new PR for `Make Stateless component with MobX model/store`?